plugins/catalog/src/api/CatalogClient.ts

```diff
@@ -68,17 +68,13 @@ export class CatalogClient implements CatalogApi {
   ): Promise<Entity[]> {
```

Comment:
Realized we should change the signature of this now perhaps, more `getEntities({filters: [{}]})` kinda thing. Not this PR tho

Reply:
Yeah at the same time as we get the paging in, most likely